installer-tests/fail-on-32-bit-system.t

```diff
@@ -2,7 +2,7 @@ Title: Can't install Sandstorm on old kernels
 Vagrant-Box: debian-7.8-32-nocm
 
 $[run]CURL_USER_AGENT=testing /vagrant/install.sh
-$[slow]Sorry, the Sandstorm server currently only runs on x86_64 machines.
-*** INSTALLATION FAILED ***
-Report bugs at: http://github.com/sandstorm-io/sandstorm
+$[slow]*** INSTALLATION FAILED ***
+Sorry, the Sandstorm server currently only runs on x86_64 machines.
+bugs at: http://github.com/sandstorm-io/sandstorm
```

Comment:
Yeah - this way it's a more lenient check - this test system tests for
substring presence, not strict equivalence.

The start of the string changed, so rather than hard-code the new start of
the string, I just stopped asserting its presence.